### ISSUE TITLE
Speed

### ISSUE THREAD

**Issue by alexmigf** [zorgzerg/brackets-minimap#9]
Hi,

I add the minimap activated but when coding above 1500 lines Brackets freezes and it's difficult to code with some minor pauses between.

When i deactivate the minimap everything goes normal again.

I have 8GB of ram and i'm using normally no more then 2GB.

**Comment by zorgzerg** [comment]
Fixed. Please update to last version 3.2.0

**Comment by alexmigf** [comment]
It's giving an error after the update, i removed and tried to install again and gives error too. Need the Brackets last update?

**Comment by zorgzerg** [comment]
Hmmm... And what the error? Screenshot please? or copy-paste

**Comment by alexmigf** [comment]
Attached
![captura de ecra de 2015-03-11 11 48 11](https://cloud.githubusercontent.com/assets/533273/6595746/e6a8d918-c7e4-11e4-9aeb-3f46f058dcb5.png)
![captura de ecra de 2015-03-11 11 48 52](https://cloud.githubusercontent.com/assets/533273/6595745/e6a73e3c-c7e4-11e4-8c95-37a52689ebe6.png)

**Comment by zorgzerg** [comment]
Please, press F12, and click console tab. Find error text (red color) and
give it to me
11 марта 2015 г. 17:51 пользователь "alexmigf" notifications@github.com
написал:

**Comment by alexmigf** [comment]
Failed to load resource file:///home/alexandre/.config/Brackets/extensions/user/brackets-minimap/Minimap.js
[Extension] failed to load /home/alexandre/.config/Brackets/extensions/user/brackets-minimap - Module does not exist: file:///home/alexandre/.config/Brackets/extensions/user/brackets-minimap/Minimap.js /utils/ExtensionLoader.js:220
errback /utils/ExtensionLoader.js:220
NodeSocketTransport - start

**Comment by zorgzerg** [comment]
Ohhh... I'm so sory ((( Case sensitivity! 

Please rename 
/home/alexandre/.config/Brackets/extensions/user/brackets-minimap/MiniMap.js to /home/alexandre/.config/Brackets/extensions/user/brackets-minimap/Minimap.js 

I'l fix it into next release.

**Comment by alexmigf** [comment]
Ok, thanks!

**Comment by zorgzerg** [comment]
Please report about perfomance now

**Comment by d-akara** [comment]
Performance still slows down quite a bit when enabled.
As a test, I'm looking at jQuery-2.1.3.js.  With minimap enabled, everything gets very laggy trying to move around the file.

**Comment by zorgzerg** [comment]
Yeah ... for very large files minimap works terribly slowly, unfortunately. But I have an idea how it can be fixed. Please wait a little longer, I hope I will be able to solve this problem. 

Thx for report
